src/kOS.Safe/Compilation/ProgramBuilder.cs

```diff
@@ -99,7 +99,7 @@ protected virtual void AddEndOfProgram(CodePart linkedObject, bool isMainProgram
             else
             {
                 linkedObject.MainCode.Add(new OpcodePush(0)); // all Returns now need a dummy return value on them.
-                linkedObject.MainCode.Add(new OpcodeReturn());
+                linkedObject.MainCode.Add(new OpcodeReturn(0));
```

Comment:
we are now not using the parameterless constructor for `OpcodeReturn` i would remove it but there is a comment to not do so.